dreampy_dns.py

```diff
@@ -76,13 +76,13 @@ def del_dns_record(protocol='ip'):
     if protocol == 'ipv6':
         rec_type = 'AAAA'
         record = DNS_IPV6
-        print(record)
-        print(DNS_IPV6)
+        # print(record)
```

Comment:
# Delete all print() altogether. It's most probably a remnant from my early debug sessions.

Reply:
Sounds good